webui/react/src/ioTypes.ts

```diff
@@ -319,11 +319,20 @@ export const ioLog = io.type({
   message: io.string,
   time: optional(io.string),
 });
+export const ioTrialLog = io.type({
+  id: io.string,
+  level: optional(ioLogLevelType),
+  message: io.string,
```

Comment:
**question:** is `meta: optional(io.string),` needed here? Since I remember seeing it in `src/types`.

Reply:
That's a good point. I see API response never returning `meta`, `level` and `time` fields.
So we can either:

1. Remove all 3 from TS `ioTrialLog` and `TrialLog` (but level is mentioned a couple of time in LogViewerTimestamp.tsx).
2. Remove just `meta` and `time` from both TS.
3. Add `meta` to `ioTrialLog`.

?

Reply:
(btw changed added `meta: optional(io.string)` for now)

Reply:
yep just leave it as optional, I'm checking in with @stoksc and this will be something in the near future. Hoping at least `time` and `level` will reliably get returned.

Reply:
**chore:** sorry, after further discussion, `meta` will not serve any purpose even in the future, so do you mind getting rid of that in both `ioTypes` and `types`? 🙏

Reply:
Done (will push soon).